Documentation/building/unix-instructions.md

```diff
@@ -59,3 +59,19 @@ If you see errors along the lines of `SendFailure (Error writing headers)` you m
 ```sh
 mozroots --import --sync
 ```
+
+## Steps to Build on Ubuntu
+
+*Note: verified on Ubuntu 14.04 LTS*
+
+1. Install git `sudo apt-get install git`
+2. Clone the corefx repo `git clone https://github.com/dotnet/corefx.git`
+3. Install curl-dev `sudo apt-get install libcurl4-openssl-dev`
```

Comment:
@mellinoe It looks like you are right, changing to clang 3.5 no longer requires the symlinks.  I verified this on a fresh Ubuntu 14.04 build.  I've made the necessary doc changes and added them in the most recent commit.  Let me know if you want to see any other changes! :smile: